src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/tests/latest/test_acs_commands.py

```diff
@@ -5,31 +5,43 @@
 
 import os
 import tempfile
+import unittest
 
 from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer,
                                RoleBasedServicePrincipalPreparer)
 
 # flake8: noqa
 
+AZURE_TEST_RUN_LIVE = bool(os.environ.get('AZURE_TEST_RUN_LIVE'))
+
+
+@unittest.skip("needs explicit service principal management")
```

Comment:
This will skip the entire test class in both Travis run and live run. Is this what you want?

Reply:
Yes, for this test case we should skip it right now. Here's the reasoning:
- It was using the implicitly created service principal that we cache under ~/.azure/acsServicePrincipal.json
- that SP wasn't being cleaned up by the tests
- we are all way too familiar with the problems that can happen with ~250 SPs attached to an account: your subscription is basically paralyzed and the items aren't deletable by reasonable means
- for historical reasons, the explicit --service-principal flag isn't supported by `az acs create --orchestrator-type=Kubernetes` so the decorators we use in the other test cases won't work

Until I figure out a better approach, it seems safer not to leak Service Principals with every test run.